src/json-api/json-api-module.ts

```diff
@@ -297,9 +301,28 @@ export class JsonApiModule {
       latestCommit,
       latestSuccessfullyDeployedCommit,
       latestActivityTimestamp: branch.latestActivityTimestamp,
+      token: this.tokenGenerator.branchToken(project.id, branch.name),
     };
   }
 
+  public async getLatestDeploymentIdForBranch(projectId: number, branchName: string) {
```

Comment:
Should this be `getLatestSuccessfulDeploymentIdForBranch` (and the same thing below)?